sql/pg_duckdb--0.3.0--1.0.0.sql

```diff
@@ -725,4 +727,6 @@ RETURNS TEXT
 SET search_path = pg_catalog, pg_temp
 LANGUAGE C AS 'MODULE_PATHNAME', 'pgduckdb_create_azure_secret';
 
-
+ALTER TABLE duckdb.extensions ADD COLUMN repository TEXT DEFAULT 'core';
+-- TODO: Maybe rename/replace "enabled" column of extensions with autoload or
+-- something similar.
```

Comment:
I feel we're never gonna see this TODO after 1.0 ships :-) Maybe we should move it to where the column is used?